packages/library/src/commonMain/kotlin/io/realm/internal/SuspendableNotifier.kt

```diff
@@ -138,6 +139,27 @@ internal class SuspendableNotifier(private val owner: Realm, private val dispatc
         }
     }
 
+    /**
+     * Listen to changes to a RealmList through a [Flow]. If the list is deleted, `null` is emitted and the flow will complete.
```

Comment:
This comment doesn't match the method signature as it is non-nullable. Not 100% sure which of them needs to change.

Reply:
If I remember correctly, we changed it to the Flow just being closed instead of emitting `null`

Reply:
Makes sense. It looks like it was scaffolding copy/paste from the method above for listening to Realm objects 🙊